hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/info/NetworkInfo.java

```diff
@@ -31,4 +32,20 @@ public interface NetworkInfo {
      */
     @NonNull
     Bytes ledgerId();
+
+    /**
+     * Gets the current deployed version of software on the network
+     *
+     * @return The deployed version
+     */
+    @NonNull
+    SemanticVersion servicesVersion();
+
+    /**
+     * Get the current deployed HAPI version running on the network.
+     *
+     * @return The HAPI version
+     */
+    @NonNull
+    SemanticVersion hapiVersion();
```

Comment:
If we're using config to get the version information (as I see is the case in NetworkGetVersionINfoHandler) then I guess we don't need this yet?

Reply:
Yep. Deleted changes.